lib/optimizely/decision_service.rb

```diff
@@ -173,10 +174,7 @@ def get_variation_for_feature_experiment(feature_flag, user_id, attributes = nil
                 Logger::INFO,
                 "The user '#{user_id}' is bucketed into experiment '#{experiment_key}' of feature '#{feature_flag_key}'."
             )
-            return {
-                'experiment' => experiment,
-                'variation' => variation
-            }
+            return Decision.new(experiment, variation, DECISION_SOURCE_EXPERIMENT)
```

Comment:
Update method return def for this method. The method returns Decision struct and not a hash

Reply:
Still pending